fenix/app/src/test/java/org/mozilla/fenix/components/metrics/InstallReferrerMetricsServiceTest.kt

```diff
@@ -0,0 +1,110 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Nice tests 👍 Nit: it may also be useful to include explicit tests for `isEmpty`, `shouldTrack`, `stop` & `start`